marklogic-data-hub-central/ui/src/components/sidebar/sidebar.tsx

```diff
@@ -34,6 +34,7 @@ interface Props {
   setHubArtifactsVisibilityPreferences: any;
   hideDataHubArtifacts: boolean;
   cardView: boolean;
+  graphView: boolean;
```

Comment:
Are we using this boolean?
Can we use it in line 690? Show related with we have related and we are in graphView.
Also in line 91. Do not even process related if we are not in graph view.

Reply:
@aniamy Good find. I brought it in for that exact reason but planned to continue that effort as part of DHFPROD-8433 because I thought this PR was a higher priority. Since the PR hasn't gone through yet, I'll include the logic here.